.github/workflows/ci.yml

```diff
@@ -19,6 +19,8 @@ jobs:
                     -   php: "8.2"
                     -   php: "8.3"
                     -   php: "8.4"
+                    -   php: "8.4"
+                        composer-stability: dev
```

Comment:
it still installs symfony/process v7
https://github.com/PHP-CS-Fixer/cli-executor/actions/runs/19081571203/job/54521925562#step:5:86

Reply:
Yes, because of PHP-CS-Fixer itself: 
```
➜  php-cs-fixer-cli-executor git:(symfony-8) ✗ sfcp why-not symfony/process ^8.0
friendsofphp/php-cs-fixer v3.89.1 requires symfony/process (^5.4.47 || ^6.4.24 || ^7.2) 
```

Couldn't we use the PHP-CS-Fixer shim?

Reply:
we can

simply without showing v8 on CI, we gamble if this change works